app/app-config.yaml

```diff
@@ -11,7 +11,7 @@ app:
             title: GitHub Issues
 
 organization:
-  name: 'Azure Development Platform: '
+  name: "Azure Development Platform: "
```

Comment:
Could you run `yarn prettier` and `yarn lint:all --fix`, these should be single quotes according to prettier. Theres a few other files which get updated by it too

Reply:
I think we need to upgrade the version of `@backstage/cli` too. We currently have 2 different versions in use and that is causing the linting to fail. Also give `yarn backstage-cli versions:check --fix` a run